projects/scion/microfrontend-platform.e2e/src/context/context-list.po.ts

```diff
@@ -31,9 +31,7 @@ export class ContextListPO {
     const contextListItemPOs: ContextEntryListItemPO[] = await this.getContextListItemPOs(waitUntil);
 
     const map = new Map<string, any>();
-    // do not use 'for-of' loop as 'for await...of' doesn't work with async iterators that are not async iterables.
-    for (let i = 0; i < contextListItemPOs.length; i++) {
-      const listItemPO = contextListItemPOs[i];
+    for (const listItemPO of contextListItemPOs) {
```

Comment:
The comment suggested that `for wait ... of` does not work. Since we are not awaiting inside statement itself but rather the for-block, I think it works.
Or do you prefer to revert this change?

Reply:
I just have a bad feeling because I vaguely remember it not working when we were waiting in the block. But if it didn't work, some tests would fail, right?

Reply:
As discussed we leave it like this, as the test would fail if it didn't work.